src/main.cpp

```diff
@@ -2,14 +2,17 @@
 // Copyright (c) 2020-2021, Athena Parthenon Collaboration. All rights reserved.
 // Licensed under the 3-Clause License (the "LICENSE");
 
+#include <sstream>
+
 // Parthenon headers
 #include "globals.hpp"
 #include "parthenon_manager.hpp"
 
 // AthenaPK headers
 #include "hydro/hydro.hpp"
 #include "hydro/hydro_driver.hpp"
-#include "main.hpp"
+//#include "main.hpp"
```

Comment:
Why did you comment this?
A `XYZ.cpp` should generally include its `XYZ.hpp` header.

Reply:
Yes but it was not using any symbols from hpp file directly. If you want I can uncomment it

Reply:
Yes, our headers could definitely be cleaned up.
We have not been too diligent to separate private and public headers (yet).

Reply:
@pgrete fixed